src/main/java/uk/gov/hmcts/probate/service/FileTransferService.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.probate.service;
+
+import feign.Response;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import uk.gov.hmcts.probate.exception.BadRequestException;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+@Slf4j
+public class FileTransferService {
+
+    private final FileTransferApi fileTransferApi;
+    private static final String VALID_FROM = "2018-03-28";
+    private static final String SS = "b";
+    private static final String SRT = "sco";
+    private static final String SP = "rwdlac";
```

Comment:
do we know what these codes are for? perhaps we can use a more descriptive name for them

Reply:
not a single clue, i asked James what they were for and it seems that they dont know either, i apparently just need them, which i do. however i tried to include some descriptive names from what i interpreted such as the valid from date which should have been `sv`. unfortunately they couldnt inform me of their purpose so i dont know what to call them other than their paramter value